DigitalLearningSolutions.Data.Tests/Services/PasswordResetServiceTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using System.Data;
+    using Dapper;
+    using DigitalLearningSolutions.Data.Exceptions;
+    using DigitalLearningSolutions.Data.Models;
+    using DigitalLearningSolutions.Data.Models.Email;
+    using DigitalLearningSolutions.Data.Models.User;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.Helpers;
+    using FakeItEasy;
+    using Microsoft.Extensions.Logging;
+    using NUnit.Framework;
+
```

Comment:
Looks like there's some unused imports - I recommend running ReSharper's CleanUpCode on the files you touched to remove unused imports & use specific formatting (I'll ignore any potential formatting issues for the rest of the review) :)